dist/stat_data.py

```diff
@@ -879,6 +879,32 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_search_near_prefix_fast_paths', 'Total number of times a search near has exited due to prefix config'),
     CursorStat('cursor_skip_hs_cur_position', 'Total number of entries skipped to position the history store cursor'),
 
+    ##########################################
+    # Cursor API error statistics
+    ##########################################
+    CursorStat('cursor_bound_error', 'cursor bound calls that return an error'),
```

Comment:
I did consider that, generally I decided not to because it would double / triple etc the number of statistics depending on the number of cursors we decided to support. In general table cursors call into the underlying file cursors, so you'd see a statistic increment for the file cursor and simultaneously you'd sees an increment on the table cursor. I think for now I'm happy with it is as is.

Reply:
Cool, that makes sense. Thanks for the explanation